alfresco-transform-tika/alfresco-transform-tika/pom.xml

```diff
@@ -49,6 +49,10 @@
                     <groupId>org.quartz-scheduler</groupId>
                     <artifactId>quartz</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
Should this be a separate PR - does not seem related to ImageMagick tests. See ATS-692 ?